develop/schemadoc/Protocol/Protocol.QActions.QAction-dllImport.md

```diff
@@ -16,47 +16,47 @@ Specifies the names of DLL files used by the QAction.
 
 ## Remarks
 
-Multiple values have to be separated by semicolons (”;”).
+Multiple values have to be separated by semicolons (";").
 
-When a DLL provided by the Microsoft .NET Framework is used (e.g. System.Xml.dll), it is sufficient to just mention it in the dllImport attribute. System DLL files must be stored in the system dll directory. All other DLL files must be stored in the C:\Skyline DataMiner\ProtocolScripts directory.
+When a DLL provided by the Microsoft .NET Framework is used (e.g. System.Xml.dll), it is sufficient to just mention it in the *dllImport* attribute. System DLL files must be stored in the system dll directory. All other DLL files must be stored in the C:\Skyline DataMiner\ProtocolScripts directory.
 
-The following DLL files do not have to be specified in the dllImport attribute as these are referenced by default:
+The following DLL files do not have to be specified in the *dllImport* attribute, as these are referenced by default:
 
 - System.dll
 - System.Core.dll
-- System.Xml.dll (RN 19494)
-- nestandard.dll (from DataMiner 10.1.11 onwards (RN 30755))
+- System.Xml.dll<!-- RN 19494 -->
+- nestandard.dll<!-- RN 30755 -->
 - SLManagedScripting.dll
 - SLNetTypes.dll
 - Interop.SLDms.dll
 - [ProtocolName].[ProtocolVersion].QAction.Helper.dll
 - QActionHelperBaseClasses.dll
-- Skyline.DataMiner.Storage.Types.dll (from DataMiner 10.0.6 onwards (RN 25036))
-- SLLoggerUtil.dll (from DataMiner 10.0.10 onwards (RN 26434))
+- Skyline.DataMiner.Storage.Types.dll<!-- RN 25036 -->
+- SLLoggerUtil.dll<!-- RN 26434 -->
 
 > [!NOTE]
 >
 > - When assemblies are referenced using the dllImport attribute, the C:\Skyline DataMiner\ProtocolScripts folder is searched prior to the C:\Skyline DataMiner\Files folder. This means that when both these folders contain a DLL with the specified name, the DLL in the ProtocolScripts folder will be used.
-> - From DataMiner 10.1.3 (RN 28653) onwards, it is possible to reference DLLs starting with either "System." or "Microsoft." if they do not exist as default .NET assembly but do exist in the ProtocolScripts folder. Prior to DataMiner 10.1.3, the C:\Skyline DataMiner\ProtocolScripts and C:\Skyline DataMiner\Files folders are not searched, as prior to that DataMiner version, these are considered system DLLs.
+> - In legacy DataMiner versions prior to DataMiner 10.1.3, it is not possible to reference DLLs starting with either "System." or "Microsoft." if they do not exist as default .NET assembly, even if they do exist in the ProtocolScripts folder.<!-- RN 28653 -->
```

Comment:
@PedroDebevere, could you confirm if this change is OK?

Reply:
change is ok